modular/stonekeep/code/jobs/job_types/adventurer/types/combat/barbarian.dm

```diff
@@ -52,13 +52,13 @@
 	switch(weapontype)
 		if("Sword")
 			beltr = /obj/item/weapon/sword/iron
-			H.mind?.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
+			H.mind?.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)//to guarantee average skill on a weapon + skilled in another
 		if("Club")
-			beltr = /obj/item/weapon/mace/woodclub
-			H.mind?.adjust_skillrank(/datum/skill/combat/axesmaces, 2, TRUE)
+			beltr = /obj/item/weapon/mace/spiked//buffed to a iron mace
```

Comment:
Honestly based, beyond roleplay there was no point in such a fucking subpar weapon you could craft regardless